### PR TITLE
Resolves #643

### LINKED ISSUE #8: Display Currently Available Bounties
We should have a filter that indicates the current assignment status of a bounty. This way, bounty hunters can filter the view for only available bounties. It could be annoying and tedious if all the bounties they are interested in handling are taken. It takes a few clicks right now to view the status of the bounty.

Proposed solutions:

1. Directly assign the assignee in the `devpool-directory` mirror of the issue. This may lead to unnecessary notifications (or not!) but it will be the most direct way to express that the bounty is taken.
2. Add a label on bounties that are currently assigned e.g. `Unavailable`

Both solutions will allow bounty hunters to filter for only available bounties.

### LINKED ISSUE #643: Display Currently Available Bounties
https://github.com/ubiquity/devpool-directory-bounties/issues/8

### PR DESCRIPTION
**Pull request by harjaapdhillon16**
Resolves https://github.com/ubiquity/devpool-directory-bounties/issues/8

I created a assignLabelToUnavailableIssues function where the function check if the issue has an assignee or not and if it does then , it just adds the label Unavailable to the issue

### CODE REVIEW

**Comment by 0x4007** [comment]
> Directly assign the assignee in the devpool-directory mirror of the issue. This may lead to unnecessary notifications (or not!) but it will be the most direct way to express that the bounty is taken.

Did you try this? Would you be able to share what might have went astray? This is the ideal setup so I just wanted to make sure that it was not possible or something.

**Comment by harjaapdhillon16** [comment]
@pavlovcik The first method of assigning assignee to the issues is totally possible and I have changed the function implementation to do that instead , I just did the 2nd implementation before because it seemed simpler to implement although I have changed it now :)

File: projects.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
         "https://github.com/ubiquity/scraper-kernel",
         "https://github.com/ubiquity/bounty-bot",
         "https://github.com/ubiquity/ubiquity-dollar",
-        "https://github.com/ubiquity/devpool-directory-bounties"
+        "https://github.com/ubiquity/devpool-directory-bounties",
+        "https://github.com/harjaapdhillon16/devpool-directory-bounty"
```
Comment (rndquu) [inline projects.json]: Why did you add your own repo? Pls remove it

File: index.ts [hunk 2]
```diff
@@ -155,3 +159,48 @@ function getRepoCredentials(projectUrl: string) {
...
+  });
+};
+
+/**
+ * Returns owner, repository names and issues number from a issue URL
+ * @param projectUrl issue URL
+ * @returns array of owner , repository names and issue number
+ */
+
+function getIssueDetailsWithLink(projectUrl: string) {
```
Comment (rndquu) [inline index.ts]: This method https://github.com/ubiquity/devpool-directory/blob/9a87b40bff54c5dc81f0205faf79f18c0ba2fba9/index.ts#L155 does pretty much the same job, pls refactor `getRepoCredentials()` and remove `getIssueDetailsWithLink ()`

File: index.ts [hunk 3]
```diff
@@ -33,6 +34,9 @@ async function main() {
         // get devpool issues
         const devpoolIssues: Issue[] = await getAllIssues(DEVPOOL_OWNER_NAME, DEVPOOL_REPO_NAME);
 
+        //to assign issues to assignee when they have been taken
+        assignAssigneeToUnavailableIssues(devpoolIssues);
```
Comment (rndquu) [inline index.ts]: @pavlovcik assigning bounty hunters to issues from some unknown repo seems to be a strange UX, besides there will be unnecessary notifications. Adding the `Unavailable` label would be a better solution. What do you think?

**Review by rndquu — CHANGES_REQUESTED** [review]

File: index.ts (hunk 3 above)
Comment (0x4007) [reply index.ts]: After trying it out I will have more confidence on what the right thing to do here is, of the two options I presented in the specification. 

I assume that notifications can be disabled on the repository if they become an issue. 

I also figured that setting the assignee on the directory copy of the issue would be the most expressive implementation. I generally like to design systems to convey more information for analytics and extensibility reasons.

**Comment by rndquu** [comment]
@harjaapdhillon16 could you show an example of how it works on your own repo?

File: index.ts (hunk 2 above)
Comment (harjaapdhillon16) [reply index.ts]: done

File: projects.json (hunk 1 above)
Comment (harjaapdhillon16) [reply projects.json]: done

**Comment by harjaapdhillon16** [comment]
> @harjaapdhillon16 could you show an example of how it works on your own repo?


https://github.com/ubiquity/devpool-directory/assets/56120762/ad74681c-56e5-442a-bff2-084e8a7a00b8



Here in the video what we are doing is basically we have 2 issues , issue 1 and issue 2 and issue 1 is assigned to me and in the body of issue 2 we have mentioned issue 1 , so the code when we run it , it automatically assigns issue 2 to me by checking the assignee in issue 1

**Comment by harjaapdhillon16** [comment]
@rndquu You can review this again ! I have made the requested changes

**Review by rndquu — CHANGES_REQUESTED** [review]
> > @harjaapdhillon16 could you show an example of how it works on your own repo?
> 
>  Untitled.mp4 
> Here in the video what we are doing is basically we have 2 issues , issue 1 and issue 2 and issue 1 is assigned to me and in the body of issue 2 we have mentioned issue 1 , so the code when we run it , it automatically assigns issue 2 to me by checking the assignee in issue 1

1. In the provided example you call API to self assign the issue to yourself. It means that you are using your own github PAT, not the github app's token. Could you add the bot's github app to your repo and show how the test bot assigns issues?
2. Could you provide an example of how the bot assigns **some other account** (not your own) ?

**Comment by harjaapdhillon16** [comment]
https://github.com/ubiquity/devpool-directory/assets/56120762/f7a9abfe-f6b1-460d-bd39-560d06b9a48c

@rndquu As you can see here I assign the issue to another account , meaning I am using the github app's token

**Comment by rndquu** [comment]
> Untitled.1.mp4 
> @rndquu As you can see here I assign the issue to another account , meaning I am using the github app's token

How to test:
1. Create a test github organization
2. Add the bot to that organization
3. `git clone https://github.com/ubiquity/devpool-directory` to your test org
4. Create a new repo with bounties
5. Let `devpool-directory` sync issues with the bounties repo from step 4
6. Assign one of the bounties (in the repo created at step 4) to your test user `test-account-2023` **who should not be a member of your test organization**
7. Let `devpool-directory` sync again and update assignees

This would be a great example

I'm this picky because from my point of view the bot shouldn't have enough permissions to assign bounty hunters who are not part of the organization. That is why I'm asking for a live example

**Comment by harjaapdhillon16** [comment]
> > Untitled.1.mp4
> > @rndquu As you can see here I assign the issue to another account , meaning I am using the github app's token
> 
> How to test:
> 
> 1. Create a test github organization
> 2. Add the bot to that organization
> 3. `git clone https://github.com/ubiquity/devpool-directory` to your test org
> 4. Create a new repo with bounties
> 5. Let `devpool-directory` sync issues with the bounties repo from step 4
> 6. Assign one of the bounties (in the repo created at step 4) to your test user `test-account-2023` **who should not be a member of your test organization**
> 7. Let `devpool-directory` sync again and update assignees
> 
> This would be a great example
> 
> I'm this picky because from my point of view the bot shouldn't have enough permissions to assign bounty hunters who are not part of the organization. That is why I'm asking for a live example

Which BOT are you referring to here , I don't understand !

**Comment by rndquu** [comment]
> > > Untitled.1.mp4
> > > @rndquu As you can see here I assign the issue to another account , meaning I am using the github app's token
> > 
> > 
> > How to test:
> > 
> > 1. Create a test github organization
> > 2. Add the bot to that organization
> > 3. `git clone https://github.com/ubiquity/devpool-directory` to your test org
> > 4. Create a new repo with bounties
> > 5. Let `devpool-directory` sync issues with the bounties repo from step 4
> > 6. Assign one of the bounties (in the repo created at step 4) to your test user `test-account-2023` **who should not be a member of your test organization**
> > 7. Let `devpool-directory` sync again and update assignees
> > 
> > This would be a great example
> > I'm this picky because from my point of view the bot shouldn't have enough permissions to assign bounty hunters who are not part of the organization. That is why I'm asking for a live example
> 
> Which BOT are you referring to here , I don't understand !

This one https://github.com/ubiquity/ubiquibot which creates [new bounties](https://github.com/ubiquity/devpool-directory/issues/666#issue-1745782540) and is supposed to handle assignments

**Comment by harjaapdhillon16** [comment]
While following this flow I am kind of facing problems while integrating the bot into my test-organization , can you provide a source where it is explained on how to do that or can you show that to me instead , so that i'll be able to do it and show the functionality ! @rndquu

**Comment by rndquu** [comment]
> While following this flow I am kind of facing problems while integrating the bot into my test-organization , can you provide a source where it is explained on how to do that or can you show that to me instead , so that i'll be able to do it and show the functionality ! @rndquu

You should:
1. Create a test github organization
2. Setup the bot [locally](https://github.com/ubiquity/ubiquibot#how-to-run-locally)
3. Add the bot to your test organization
4. `git clone https://github.com/ubiquity/devpool-directory` to your test org
5. In your forked `devpool-directory` set [app_id](https://github.com/ubiquity/devpool-directory/blob/development/.github/workflows/sync-issues.yml#L30) and [private_key](https://github.com/ubiquity/devpool-directory/blob/development/.github/workflows/sync-issues.yml#L31) with your local bot instance params (you can find them in the bot's github app settings)
6. Create a new repo with bounties
7. Let `devpool-directory` sync issues with the bounties repo from step 6
8. Assign one of the bounties (in the repo created at step 6) to your test user `test-account-2023` **who should not be a member of your test organization**
9. Let `devpool-directory` sync again and update assignees

**Comment by harjaapdhillon16** [comment]
I tried to do it and the authentication isn't going though and I am getting an error when trying to update assignees @rndquu 

https://github.com/ubiquity/devpool-directory/assets/56120762/971c8583-9c16-40b2-8dcf-9f300f15b3a7

**Comment by rndquu** [comment]
> I tried to do it and the authentication isn't going though and I am getting an error when trying to update assignees @rndquu
> 
>  Untitled.2.mp4

could you provide a full error log ?

**Comment by harjaapdhillon16** [comment]
```log
RequestError [HttpError]: Validation Failed: {"value":"Draeieg","resource":"Issue","field":"assignees","code":"invalid"}
    at <anonymous> (/Users/harjaapdhillon/Desktop/web3/devpool-directory-latest/node_modules/@octokit/request/dist-node/index.js:122:21)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at requestWithGraphqlErrorHandling (/Users/harjaapdhillon/Desktop/web3/devpool-directory-latest/node_modules/@octokit/plugin-retry/dist-node/index.js:70:20)
    at Job.doExecute (/Users/harjaapdhillon/Desktop/web3/devpool-directory-latest/node_modules/bottleneck/light.js:405:18) {
  status: 422,
  response: {
    url: 'https://api.github.com/repos/test-organization-devpool/devpool-directory/issues/61',
    status: 422,
    headers: {
      'access-control-allow-origin': '*',
      'access-control-expose-headers': 'ETag, Link, Location, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Used, X-RateLimit-Resource, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval, X-GitHub-Media-Type, X-GitHub-SSO, X-GitHub-Request-Id, Deprecation, Sunset',
      'content-length': '202',
      'content-security-policy': "default-src 'none'",
      'content-type': 'application/json; charset=utf-8',
      date: 'Mon, 12 Jun 2023 22:23:09 GMT',
      'github-authentication-token-expiration': '2023-07-12 02:56:14 +0530',
      'referrer-policy': 'origin-when-cross-origin, strict-origin-when-cross-origin',
      server: 'GitHub.com',
      'strict-transport-security': 'max-age=31536000; includeSubdomains; preload',
      vary: 'Accept-Encoding, Accept, X-Requested-With',
      'x-content-type-options': 'nosniff',
      'x-frame-options': 'deny',
      'x-github-api-version-selected': '2022-11-28',
      'x-github-media-type': 'github.v3; format=json',
      'x-github-request-id': 'ED0B:77DA:5C4F1D2:BD62796:64879ACD',
      'x-ratelimit-limit': '5000',
      'x-ratelimit-remaining': '4901',
      'x-ratelimit-reset': '1686612005',
      'x-ratelimit-resource': 'core',
      'x-ratelimit-used': '99',
      'x-xss-protection': '0'
    },
    data: {
      message: 'Validation Failed',
      errors: [
        {
          value: 'Draeieg',
          resource: 'Issue',
          field: 'assignees',
          code: 'invalid'
        }
      ],
      documentation_url: 'https://docs.github.com/rest/reference/issues#update-an-issue'
    }
  },
  request: {
    method: 'PATCH',
    url: 'https://api.github.com/repos/test-organization-devpool/devpool-directory/issues/61',
    headers: {
      accept: 'application/vnd.github.v3+json',
      'user-agent': 'octokit.js/2.0.19 octokit-core.js/4.2.1 Node.js/18.16.0 (darwin; x64)',
      authorization: 'token [REDACTED]',
      'content-type': 'application/json; charset=utf-8'
    },
    body: '{"assignees":["Draeieg"]}',
    request: { hook: [Function: bound bound register] }
  }
}
```
@rndquu

**Comment by 0x4007** [comment]
Hey @harjaapdhillon16 thanks for your investigative work. I asked ChatGPT and I believe that the assignment approach might not be possible.

Let's verify the following checklist to be sure:

### ChatGPT
From the error message, it seems like the username "Draeieg" is not a valid assignee for the issue. This could be due to a number of reasons:

- [ ] 1. **The username doesn't exist.** Make sure the spelling and capitalization of the username are correct. 

- [x] 2. **The user is not a collaborator on the repository.** In order to assign a user to an issue, they need to be a collaborator on the repository. You can invite them to be a collaborator from the repository's settings.

- [x] 3. **The user hasn't accepted the collaborator invitation.** Even if you've sent an invitation to the user to become a collaborator, they need to accept the invitation before they can be assigned to issues.

- [x] 4. **The user doesn't have write access to the repository.** In order to be assigned to an issue, the user needs to have write access to the repository.

Please verify the above points to ensure you can assign the user to the issue. If everything seems correct but you're still encountering the error, you may want to check with GitHub's support or the Octokit library maintainers for additional help.

---

I believe that Draeieg does not have write access to your fork because they have not been added as a collaborator. This means that this system will not work for any bounty hunter that is not already added as a collaborator to the DevPool Directory. This means we will need to use `unavailable`. Or if we want to be more expressive, we can create a label that says `assignee: ${username}` do you think you can do that?

**Comment by harjaapdhillon16** [comment]
yeah surely i can do it , @rndquu should I make the change now ?

**Comment by rndquu** [comment]
> at Job.doExecute (/Users/harjaapdhillon/Desktop/web3/devpool-directory-latest/node_modules/

yes, it would be great if you could refactor the code and use the `Unavailable` label approach

**Comment by harjaapdhillon16** [comment]
Cool that's what I had done before ! I'll go back to that !

**Comment by harjaapdhillon16** [comment]
@rndquu Rolled back to the previous implementation

**Review by rndquu — CHANGES_REQUESTED** [review]
There are 2 issues in the current implementation:
1. The `Unavailable` label is assigned [here](https://github.com/ubiquity/devpool-directory/blob/cce280810bfed873ca2f44e60d9e66dafa514b2c/index.ts#L38) **before** title/state/price labels are updated [here](https://github.com/ubiquity/devpool-directory/blob/cce280810bfed873ca2f44e60d9e66dafa514b2c/index.ts#L53) and [here](https://github.com/ubiquity/devpool-directory/blob/cce280810bfed873ca2f44e60d9e66dafa514b2c/index.ts#L70). It means that the `Unavailable` label will be removed if any of an issue's title/state/price values have been updated. Have you tested the code by updating an issue's title/state/price values?
2. In the current implementation we call `issue.update()` 2 times: [here](https://github.com/ubiquity/devpool-directory/blob/cce280810bfed873ca2f44e60d9e66dafa514b2c/index.ts#L53) and [here](https://github.com/ubiquity/devpool-directory/blob/cce280810bfed873ca2f44e60d9e66dafa514b2c/index.ts#L183). We'd better optimize to update an issue in a single API call because we are already near the github rate limits.

What can be done better:
1. If an issue is a new one then [here](https://github.com/ubiquity/devpool-directory/blob/cce280810bfed873ca2f44e60d9e66dafa514b2c/index.ts#L75) we could check an issue's assignee and set the `Unavailable` label if there is no assignee
2. If issue already exists in the https://github.com/ubiquity/devpool-directory then [here](https://github.com/ubiquity/devpool-directory/blob/cce280810bfed873ca2f44e60d9e66dafa514b2c/index.ts#L52) we could check if assignee has changed since the last check (i.e. if there is an assignee then add the `Unavailable` label else remove it)

**Comment by harjaapdhillon16** [comment]
@rndquu I made the fixes according to your comments , review this and lemme know !

**Comment by harjaapdhillon16** [comment]
The issue is being updated if it exists with a Unavailable label, and if it is being created it gets created with an Unavailable label

File: index.ts [hunk 4]
```diff
@@ -153,5 +165,6 @@ function getRepoCredentials(projectUrl: string) {
     const urlPath = urlObject.pathname.split('/');
     const ownerName = urlPath[1];
     const repoName = urlPath[2];
-    return [ownerName, repoName];
-}
+    const issueNumber = urlPath?.[4];
```
Comment (rndquu) [inline index.ts]: this `issueNumber` is not used anywhere and can be removed

File: index.ts [hunk 5]
```diff
@@ -63,14 +70,19 @@ async function main() {
                     // if issue is "closed" then skip it, no need to copy/paste already "closed" issues
                     if (projectIssue.state === 'closed') continue;
                     // create a new issue
+                    const issueDetails = await octokit.rest.issues.get({
```
Comment (rndquu) [inline index.ts]: here `projectIssue = issueDetails` so we don't need to call API to get an issue info 1 more time, simply use `projectIssue`

File: index.ts [hunk 6]
```diff
@@ -44,6 +45,12 @@ async function main() {
                 // if issue exists in devpool
                 const devpoolIssue = getIssueByLabel(devpoolIssues, `id: ${projectIssue.node_id}`);
                 if (devpoolIssue) {
+                    const issueDetails = await octokit.rest.issues.get({
```
Comment (rndquu) [inline index.ts]: `projectIssue = issueDetails`, no need for additional API call, simply use `projectIssue`

File: index.ts [hunk 7]
```diff
@@ -44,6 +45,12 @@ async function main() {
...
+                        owner: ownerName,
+                        repo: repoName,
+                        issue_number: devpoolIssue.number,
+                    });
+                    const additionalLabelsToAdd = issueDetails.data?.assignee ?['Unavailable']:[]
                     // if title or state or pricing was updated then update a devpool issue
                     if (devpoolIssue.title !== projectIssue.title || devpoolIssue.state !== projectIssue.state || getIssuePriceLabel(devpoolIssue) !== getIssuePriceLabel(projectIssue)) {
```
Comment (rndquu) [inline index.ts]: if assignee has been updated then the `Unavailable` label will appear/disappear only if title/state/price labels have changed, i.e. right now on assignee change the issue is not updated, pls fix

**Review by rndquu — CHANGES_REQUESTED** [review]

File: index.ts (hunk 5 above)
Comment (harjaapdhillon16) [reply index.ts]: fixed

File: index.ts (hunk 4 above)
Comment (harjaapdhillon16) [reply index.ts]: fixed

File: index.ts (hunk 6 above)
Comment (harjaapdhillon16) [reply index.ts]: fixed

File: index.ts (hunk 7 above)
Comment (harjaapdhillon16) [reply index.ts]: fixed

**Comment by harjaapdhillon16** [comment]
@rndquu Made the changes you requested

File: index.ts [hunk 8]
```diff
@@ -29,55 +50,78 @@ const octokit = new Octokit({ auth: process.env.DEVPOOL_GITHUB_API_TOKEN });
...
+        );
+        if (devpoolIssue) {
+          const additionalLabelsToAdd = projectIssue?.assignee?.login
+            ? ["Unavailable"]
+            : [];
+          // if title or state or pricing was updated then update a devpool issue
+          if (
+            devpoolIssue.title !== projectIssue.title ||
+            devpoolIssue.state !== projectIssue.state ||
+            devpoolIssue.assignee?.login !== projectIssue?.assignee?.login ||
```
Comment (rndquu) [inline index.ts]: This is really hard to review when the whole code is formatted

The `Unavailable` label won't be removed if the partner project's issue assignee is unassigned from the issue because in this case `devpoolIssue.assignee?.login !== projectIssue?.assignee?.login` returns `false`, pls fix

**Review by rndquu — CHANGES_REQUESTED** [review]

File: index.ts (hunk 8 above)
Comment (harjaapdhillon16) [reply index.ts:81]: I think formatting was necessary so I lint formatted it ! And I also have added the condition where unassigned will be fixed

Comment (harjaapdhillon16) [reply index.ts]: Apologies for the formatting removed it and also added this condition !

**Comment by harjaapdhillon16** [comment]
@rndquu You can review again !

File: index.ts [hunk 9]
```diff
@@ -18,6 +18,27 @@ type Issue = {
     pull_request: null | {},
     state: 'open' | 'closed',
     title: string,
+    body?: string;
+    assignee: {
+      login: string;
+      id: number;
+      node_id: string;
+      avatar_url: string;
+      gravatar_id: string;
```
Comment (rndquu) [inline index.ts]: pls remove unnecessary fields which are not used in the code

File: index.ts [hunk 10]
```diff
@@ -44,15 +68,18 @@ async function main() {
                 // if issue exists in devpool
                 const devpoolIssue = getIssueByLabel(devpoolIssues, `id: ${projectIssue.node_id}`);
                 if (devpoolIssue) {
+                  const additionalLabelsToAdd = projectIssue?.assignee?.login
+                    ? ["Unavailable"]
+                    : [];
                     // if title or state or pricing was updated then update a devpool issue
-                    if (devpoolIssue.title !== projectIssue.title || devpoolIssue.state !== projectIssue.state || getIssuePriceLabel(devpoolIssue) !== getIssuePriceLabel(projectIssue)) {
+                    if (devpoolIssue.title !== projectIssue.title || devpoolIssue.state !== projectIssue.state || devpoolIssue.assignee?.login !== projectIssue?.assignee?.login || projectIssue?.assignee?.login === undefined || getIssuePriceLabel(devpoolIssue) !== getIssuePriceLabel(projectIssue)) {
```
Comment (rndquu) [inline index.ts]: `projectIssue?.assignee?.login === undefined` means that an issue will be updated **every time** if there is no assignee. Github API calls are expensive in terms of rate limits. Basically we should this condition here `if ("Unavailable" label exists and there is no partner issue's assignee OR "Unavailable" label does not exist and there is partner issue's assignee) then update an issue`

**Review by rndquu — CHANGES_REQUESTED** [review]

File: index.ts (hunk 9 above)
Comment (harjaapdhillon16) [reply index.ts]: Fixed

File: index.ts (hunk 10 above)
Comment (harjaapdhillon16) [reply index.ts]: Fixed

**Comment by harjaapdhillon16** [comment]
@rndquu You can review again !

File: index.ts [hunk 11]
```diff
@@ -71,8 +71,10 @@ async function main() {
                   const additionalLabelsToAdd = projectIssue?.assignee?.login
                     ? ["Unavailable"]
                     : [];
+                    const isUnavailableTag = projectIssue?.labels?.some((item)=>item.name==="Unavailable");
```
Comment (rndquu) [inline index.ts]: shouldn't we check devpool issue instead? Partner project issues can't have the `Uavailable` tag

```
const isUnavailableTag = devpoolIssue?.labels?.some((item)=>item.name==="Unavailable");
```

File: index.ts [hunk 12]
```diff
@@ -71,8 +71,10 @@ async function main() {
...
                     // if title or state or pricing was updated then update a devpool issue
-                    if (devpoolIssue.title !== projectIssue.title || devpoolIssue.state !== projectIssue.state || devpoolIssue.assignee?.login !== projectIssue?.assignee?.login || projectIssue?.assignee?.login === undefined || getIssuePriceLabel(devpoolIssue) !== getIssuePriceLabel(projectIssue)) {
+                    if (devpoolIssue.title !== projectIssue.title || devpoolIssue.state !== projectIssue.state ||
+                       devpoolIssue.assignee?.login !== projectIssue?.assignee?.login || (projectIssue?.assignee?.login === undefined && isUnavailableTag) || (projectIssue?.assignee?.login !== undefined && !isUnavailableTag) || getIssuePriceLabel(devpoolIssue) !== getIssuePriceLabel(projectIssue)) {
```
Comment (rndquu) [inline index.ts]: pls remove `devpoolIssue.assignee?.login !== projectIssue?.assignee?.login`

**Review by rndquu — CHANGES_REQUESTED** [review]

File: index.ts (hunk 11 above)
Comment (harjaapdhillon16) [reply index.ts]: Actailly you are right with that !

File: index.ts (hunk 12 above)
Comment (harjaapdhillon16) [reply index.ts]: Done

**Comment by harjaapdhillon16** [comment]
@rndquu You can review again

**Review by rndquu — APPROVED** [review]

**Comment by 0x4007** [comment]
@rndquu do you think it will be an issue to merge if conventional commits CI doesn't pass?

**Comment by rndquu** [comment]
> @rndquu do you think it will be an issue to merge if conventional commits CI doesn't pass?

no, it won't be an issue, @harjaapdhillon16 pls for future PRs use [conventional commits naming](https://github.com/ubiquity/ubiquity-dollar#committing-code)
